res/values/strings.xml

```diff
@@ -41,6 +41,10 @@
     <string name="main_menu">Main menu</string>
     <string name="start_chat">Start chat</string>
     <string name="show_full_message">Show full message…</string>
+    <string name="show_full_message_in_browser">Show full message in browser…</string>
+    <string name="loading">Loading</string>
```

Comment:
is that a message as "One moment…"? then maybe also add an ellipsis. in any way, i think, this strings deserves a little translator hint (just a `<!-- comment -->` atop of the string)